test/chplenv/autocomplete/autocomplete.chpl

```diff
@@ -10,12 +10,6 @@ use Spawn;
 
 var home = CHPL_HOME;
 
-extern proc getenv(name: c_string): c_string;
-extern proc setenv(name: c_string, val: c_string, overwrite: c_int): c_int;
-
-var path = getenv("PATH"):string;
-setenv("PATH", (home + "/bin/" + CHPL_HOST_PLATFORM + ":" + path).c_str(), 1);
-
```

Comment:
Why is this no longer needed?

Reply:
See commit ed0884bac20bee1aadeba08d8869bab9e4eaf8d0 -- I updated the script it calls, which seems a more reasonable place to handle the case of PATH not being set to include chpl.